content/guides/getting-started/writing-your-first-component-test.md

```diff
@@ -2,4 +2,1184 @@
 title: Writing Your First Component Test
 ---
 
-CONTENT TBD
+<!--
+<Alert type="info">
+
+## <Icon name="graduation-cap"></Icon> What you'll learn
+
+- How to start testing components in Cypress.
+- How to mount a component and test that it renders correctly.
+- How to query the DOM, stub functions, and write assertions.
+- What passing and failing tests look like.
+- How to use the Cypress UI to help debug test results.
+
+</Alert>
+
+-->
```

Comment:
Let's uncomment this so we can start to see how it makes the docs look. We can just copy once we've finalized the page's content.